apps/registration/src/components/Button.tsx

```diff
@@ -0,0 +1,31 @@
+const Button = ({
+  onClick,
+  label,
+  type,
+  className,
+  style,
+  disabled,
+}: {
+  onClick: () => void;
+  label?: string;
+  type?: string;
+  className?: string;
+  style?: string;
+  disabled?: boolean;
+}) => {
+  return (
+    <>
+      <button
+        type={type}
+        onClick={onClick}
+        className={className}
```

Comment:
Remove className props and add styling of the button according to figma krub, no need for custom styling.